spec/requests/v0/claim_documents_spec.rb

```diff
@@ -65,16 +71,18 @@
       end.not_to change(PersistentAttachment, :count)
       expect(response).to have_http_status(:unprocessable_entity)
       resp = JSON.parse(response.body)
-      expect(resp['errors'][0]['title']).to eq('File size must not be less than 1.0 KB')
+      expect(resp['errors'][0]['title']).to eq('Unprocessable Entity')
```

Comment:
This is the test for invalid pdf parameters and unable to stamp?

Reply:
It's now throwing unprocessable entity due to the file type. txt files are not processable file types. If I switch the file to use `empty-file.jpg` instead, I get a 500 error. For some reason, the mime-type of the document isn't being set for files with size of 0 bytes, and it causes an error when we try to run `.to_pdf` on such a file.

Reply:
can/should that be a separate error - 'Empty File'?  no other test files that could be used?

Reply:
We already have a guard for it on the front end, so maybe it's fine to rely on this validation?:

![image](https://github.com/user-attachments/assets/3f539ea0-4905-41bc-8a30-b027399e2f49)

Reply:
This seems to be a separate from an empty file - or at least the error message is misleading.  But that can be addressed later.  Likely will have additional tickets for modifying the message